AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.h

```diff
@@ -47,7 +58,12 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Set current auth token and account id.
  */
-- (void)setAuthToken:(NSString *)authToken withAccountId:(NSString *)accountId;
+- (void)setAuthToken:(nullable NSString *)authToken withAccountId:(nullable NSString *)accountId expiresOn:(nullable NSDate *)expiresOn;
```

Comment:
Please add comment about parameters, I would write this is a little bit more details, especially since we have new parameter here